dev/.env.micropowermanager-backend

```diff
@@ -22,3 +22,6 @@ REDIS_PASSWORD=null
 JWT_SECRET=TihQ232bmMermM8HKaLHwzRl5JSuxwqhfEAQKxRHrQqjRUwrsw1FRZhAcLkihEDU
 
 CACHE_DRIVER=redis
+# Mailer configuration for local development
+MAIL_LOG_CHANNEL=stdout
+MAIL_MAILER=log
```

Comment:
Let make this the default? I.e. by changing `config/mail.php`. Seems to be inline with Laravel default's nowadays: https://github.com/laravel/laravel/blob/11.x/config/mail.php#L17

I would also make `MAIL_LOG_CHANNEL=stdout` the default. I don't even see what other value would make sense here 😂